.changes/next-release/bugfix-AWSSDKforJavav2-17f90b1.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Added partCount and ContentRange validation for s3 transfer manager download request "
```

Comment:
The change is specific for multipart s3 client, right? Suggesting `Add additional validations for multipart download operations in the Java multipart S3 client.`